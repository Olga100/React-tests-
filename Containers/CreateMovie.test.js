import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import {CreateMovieForm} from './CreateMovie';



describe('<CreateMovieForm/>', () => {

    it('Renders input component for creating of the movie', () => {
        const component = renderer.create(
            <CreateMovieForm/>
        );
        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });


});