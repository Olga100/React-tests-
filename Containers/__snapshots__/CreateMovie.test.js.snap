// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateMovieForm/> Renders input component for creating of the movie 1`] = `
<div
  className="formContainer"
>
  <div
    className="inputContainer"
  >
    Title
    <input
      className="titleForm"
      placeholder="enter the title of the movie"
      type="text"
    />
  </div>
  <div
    className="inputContainer"
  >
    Poster
    <input
      className="titleForm"
      placeholder="enter the Url of the poster"
      type="text"
    />
  </div>
  <div
    className="inputContainer"
  >
    Genres
    <input
      className="genresForm"
      placeholder="enter the genres of the movie"
      type="text"
    />
  </div>
  <div
    className="inputContainer"
  >
    Actors
    <input
      className="actorsForm"
      placeholder="enter the actors of the movie"
      type="text"
    />
  </div>
  <div
    className="inputContainer"
  >
    Director
    <input
      className="directorForm"
      placeholder="enter the director of the movie"
      type="text"
    />
  </div>
  <div
    className="inputContainer"
  >
    Description
    <textarea
      className="descriptionForm"
      cols="30"
      name="text"
      placeholder="enter the description of the movie"
      rows="6"
    >
       
    </textarea>
  </div>
  <button
    id="submitCreateForm"
    onClick={[Function]}
    type="submit"
  >
    Create Movie
  </button>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    <button
      className="backToMovies"
      type="button"
    >
      Back to MoviesList
    </button>
  </a>
</div>
`;
